articles/azure-monitor/profiler/profiler-troubleshooting.md

```diff
@@ -50,6 +50,16 @@ Check that a firewall or proxies aren't blocking your access to [this webpage](h
 
 Profiling data is uploaded only when it can be attached to a request that happened while Profiler was running. Profiler collects data for two minutes each hour. You can also trigger Profiler by [starting a profiling session](./profiler-settings.md#profile-now).
 
+> [!NOTE]
+> Please be advised that the Profiler depends on Application Insights telemetry to detect application requests during a session. It is essential to ensure that there is Request data available; otherwise, profiler data upload will be skipped.
```

Comment:
```suggestion
> Please be advised that the Profiler depends on Application Insights telemetry to detect application requests during a session. It is essential to ensure that there is request data available; otherwise, the profiler data upload will be skipped.
```